benchmarks/Kestrel.Performance/Kestrel.Performance.csproj

```diff
@@ -23,7 +23,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.10.9" NoWarn="KRB4002" />
```

Comment:
I know we don't do it elsewhere, but maybe we should start adding a short note about what the warn-codes we're suppressing are?